tests/manage/ocs_288.py

```diff
@@ -334,36 +337,30 @@ def create_secret(file_name):
     with open(TEMP_YAML_FILE, 'w') as yaml_file:
         yaml.dump(file_y, yaml_file, default_flow_style=False)
     assert SECRET.create(yaml_file=TEMP_YAML_FILE)
-    secret_list.append(file_y.get('metadata')['name'])
+    self.secret_list.append(file_y.get('metadata')['name'])
     return True
 
 
-def create_pvc(**kwargs):
+def create_pvc(pvc_list, storageclass_list, count=1):
     """
     Function for creating pvc and multiple pvc
     :param kwargs:
     :return:
     """
-    if kwargs.get('count'):
-        pvc_count = int(kwargs.get('count'))
-    else:
-        pvc_count = 1
-    for i in range(pvc_count):
+    for i in range(count):
         sc_name = random.choice(storageclass_list)
         pvc_data = {}
         pvc_data['rbd_storageclass_name'] = sc_name
         pvc_data['pvc_name'] = "rbd-pvc-" + ''.join(
             [random.choice(string.ascii_lowercase)
              for _ in range(3)]) + str(sc_name)
-
         file_y = templating.generate_yaml_from_jinja2_template_with_data(
             PVC_RBD_YAML, **pvc_data
         )
         with open(TEMP_YAML_FILE, 'w') as yaml_file:
             yaml.dump(file_y, yaml_file, default_flow_style=False)
         assert PVC.create(yaml_file=TEMP_YAML_FILE)
         pvc_name = pvc_data.get('pvc_name')
-        pvc_name = pvc_data.get('pvc_name')
         pvc_list.append(pvc_name)
```

Comment:
Ok few things about this function.
1) It should create pvcs - if you also what to wait for resources, please add new parameter called 'wait' type bool and in case it is True, wait_for_resource()
2) if this func should create pvcs - let it return the pvc list and don't take care of maintaining 'pvc_list' which is a class attribute.

Reply:
1-:Here in this function i am creating pvc as well as checking that pvc is in Bound
2:- I will remove pvc_list and create another function to get pvc_list for teardown step